lua/competitest/commands.lua

```diff
@@ -201,11 +201,12 @@ function M.receive(mode)
 	---@param path string | function: see received_problems_path, received_contests_directory and received_contests_problems_path
 	---@param task table: table with received task data
 	---@param file_extension string
+	---@param date_format string: string used to format date
 	---@return string
-	local function eval_path(path, task, file_extension)
+	local function eval_path(path, task, file_extension, date_format)
```

Comment:
We don't need `date_format` argument here because using date for received problems and contests path is very unlikely. If someone want to do so he can configure the options `received_problems_path`, `received_contests_directory` and `received_contests_problems_path` with a function.